client/blocks/eligibility-warnings/index.tsx

```diff
@@ -38,66 +34,34 @@ interface ExternalProps {
 type Props = ExternalProps & ReturnType< typeof mergeProps > & LocalizeProps;
 
 export const EligibilityWarnings = ( {
-	backUrl,
 	context,
+	feature,
 	eligibilityData,
-	hasBusinessPlan,
 	isEligible,
-	isJetpack,
-	isVip,
 	isPlaceholder,
 	onProceed,
-	onCancel,
+	recordCtaClick,
 	siteId,
-	sitePlan,
-	siteSlug,
 	translate,
 }: Props ) => {
-	const warnings = get( eligibilityData, 'eligibilityWarnings', [] );
-
-	const listHolds = filter(
-		get( eligibilityData, 'eligibilityHolds', [] ),
-		hold => ! includes( [ 'NO_BUSINESS_PLAN', 'NOT_USING_CUSTOM_DOMAIN' ], hold )
-	);
+	const warnings = eligibilityData.eligibilityWarnings || [];
+	const listHolds = eligibilityData.eligibilityHolds || [];
 
+	const showWarnings = warnings.length > 0 && ! hasBlockingHold( listHolds );
 	const classes = classNames( 'eligibility-warnings', {
 		'eligibility-warnings__placeholder': isPlaceholder,
+		'eligibility-warnings--with-indent': showWarnings,
 	} );
 
-	let businessUpsellBanner = null;
-	if ( ! hasBusinessPlan && ! isJetpack && ! isVip ) {
-		const description = translate(
-			'Also get unlimited themes, advanced customization, no ads, live chat support, and more.'
-		);
-		const title = translate( 'Business plan required' ) as string;
-		const plan =
-			sitePlan &&
-			findFirstSimilarPlanKey( sitePlan.product_slug, {
-				type: TYPE_BUSINESS,
-			} );
-		let feature = null;
-		let event = null;
-
-		if ( 'plugins' === context ) {
-			feature = FEATURE_UPLOAD_PLUGINS;
-			event = 'calypso-plugin-eligibility-upgrade-nudge';
-		} else {
-			feature = FEATURE_UPLOAD_THEMES;
-			event = 'calypso-theme-eligibility-upgrade-nudge';
-		}
-
-		const bannerURL = `/checkout/${ siteSlug }/business`;
-		businessUpsellBanner = (
-			<Banner
-				description={ description }
-				feature={ feature }
-				event={ event }
-				plan={ plan }
-				title={ title }
-				href={ bannerURL }
-			/>
-		);
-	}
+	const logEventAndProceed = () => {
+		// We removed <Banner /> Component which logged this event on upsell click.
+		// We want to keep tracking the data in the same way, so let's fake the banner click here
+		recordCtaClick( feature );
+		onProceed();
```

Comment:
It looks like the component is not actually blocking the plugin flow (at least):

![eligibility-warning-doesnt-block](https://user-images.githubusercontent.com/1587282/71259607-d3e50900-2306-11ea-96a1-effca3e8eee3.gif)

Do we still need to build out the implementation for the click handler to launch the site, add the plan to the cart, & redirect to checkout?

Reply:
That's listed in known issues

Reply:
See https://github.com/Automattic/wp-calypso/pull/38364

Reply:
Cool.  That's all I've found -- I'll go review that PR now so we can get it merged in here.

Reply:
This is now part of this PR so it should be fully functional